tests/client/amo/reducers/testLanding.js

```diff
@@ -0,0 +1,131 @@
+import landing from 'amo/reducers/landing';
+
+describe('landing reducer', () => {
+  let initialData;
+
+  before(() => {
+    initialData = {
+      featured: { count: 0, results: [] },
+      highlyRated: { count: 0, results: [] },
+      loading: true,
+      popular: { count: 0, results: [] },
+    };
+  });
+
+  it('defaults to not loading', () => {
+    const { loading } = landing(undefined, { type: 'unrelated' });
+
+    assert.strictEqual(loading, false);
+  });
+
+  it('defaults to zero count', () => {
+    const { featured, highlyRated, popular } = landing(undefined, {
+      type: 'unrelated',
+    });
+
+    assert.strictEqual(featured.count, 0);
+    assert.strictEqual(highlyRated.count, 0);
+    assert.strictEqual(popular.count, 0);
+  });
+
+  it('defaults to empty results', () => {
+    const { featured, highlyRated, popular } = landing(undefined, {
+      type: 'unrelated',
+    });
+    assert.deepEqual(featured.results, []);
+    assert.deepEqual(highlyRated.results, []);
+    assert.deepEqual(popular.results, []);
+  });
+
+  describe('LANDING_GET', () => {
+    it('sets the initialState', () => {
+      const initialState = {
+        featured: { foo: 'bar' },
+        highlyRated: { count: 0 },
+        loading: false,
+        popular: { results: [] },
+      };
+      const {
+        addonType, featured, highlyRated, loading, popular,
+      } = landing(initialState, {
+        type: 'LANDING_GET',
+        payload: { addonType: 'theme' },
```

Comment:
I think this would be a better test if you passed `actions.loadLanding()` directly to `landing()`.